TACC/Core-Portal
# Task/FP-561 - Prevent users from being able to launch jobs with inactive allocations

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [FP-123](https://jira.tacc.utexas.edu/browse/FP-123)

## Summary of Changes: ##

## Testing Steps: ##
1. Compare the list of systems on your active allocations with what should be available with for the given job
2. run `portal.apps.users.utils.get_allocations()` with your username to get your allocations
3. Make sure that the system the job runs on is listed under the active allocations available as options.

```
from portal.apps.users.utils import get_allocations
alloc = get_allocations('yourusername')

for a in alloc['active']: 
     print('--- ' + a['title'] + ' ---') 
     for sys in a['systems']: 
         print(sys['name']) 
```

## UI Photos:
![Screen Shot 2020-08-14 at 4 40 05 PM](https://user-images.githubusercontent.com/29575979/90294772-dc1d2600-de4c-11ea-9675-481d2cafb716.png)


## Notes: ##
